include/openssl/trace.h

```diff
@@ -41,15 +41,14 @@ extern "C" {
 # define OSSL_TRACE_CATEGORY_INIT                2
 # define OSSL_TRACE_CATEGORY_TLS                 3
 # define OSSL_TRACE_CATEGORY_TLS_CIPHER          4
-# define OSSL_TRACE_CATEGORY_ENGINE_CONF         5
+# define OSSL_TRACE_CATEGORY_CONF                5
 # define OSSL_TRACE_CATEGORY_ENGINE_TABLE        6
 # define OSSL_TRACE_CATEGORY_ENGINE_REF_COUNT    7
 # define OSSL_TRACE_CATEGORY_PKCS5V2             8
 # define OSSL_TRACE_CATEGORY_PKCS12_KEYGEN       9
 # define OSSL_TRACE_CATEGORY_PKCS12_DECRYPT     10
 # define OSSL_TRACE_CATEGORY_X509V3_POLICY      11
 # define OSSL_TRACE_CATEGORY_BN_CTX             12
-# define OSSL_TRACE_CATEGORY_CONF               13
 # define OSSL_TRACE_CATEGORY_NUM                14
```

Comment:
`OSSL_TRACE_CATEGORY_NUM ` needs to be decrement.

_(BTW: wouldn't it be better to use an unnamed enumeration instead? And add a comment before `OSSL_TRACE_CATEGORY_NUM ` that it needs to be last.)_

Reply:
Fixed